src/main/java/com/fundacionjala/bankocr/BankOCR.java

```diff
@@ -0,0 +1,128 @@
+package com.fundacionjala.bankocr;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Created by Bruno on 7/7/2016.
+ */
+public class BankOCR {
+
+    private static final String INPUT_FILE_NAME = "accountsOCR.txt";
+    private static final int COLS_DIGITS = 3;
+    private static final int ROWS_DIGITS = 3;
+    private static final int NUMBER_DIGITS = 9;
+
+    private List<String> numberAccountsList = new ArrayList<String>();
```

Comment:
@rosariogarcia @sperBruno 
Don't instance in the attributes. 
Please move into constructor.